apps/core-app/src/polyfills.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Polyfill stable language features. These imports will be optimized by `@babel/preset-env`.
+ *
+ * See: https://github.com/zloirock/core-js#babel
+ */
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
```

Comment:
ah interesting. i ran into this in another vite app. was causing unexpected issues so good catch!

Reply:
I think it does this by default.